test/utils/has-state.ts

```diff
@@ -0,0 +1,9 @@
+import {expect} from 'test-drive-react';
+export default function hasState(elem: Element | null, styles: any, name: string): void | never {
```

Comment:
State is React keyword, so maybe it's better to rename this function to hasCSSState/hasCustomState/<your option>?